src/infrastructure/selectors/IdentitySelectorAdapter.spec.js

```diff
@@ -0,0 +1,37 @@
+const { describe, it, beforeEach } = require('mocha');
+const { expect } = require('chai');
+const IdentitySelectorAdapter = require('./IdentitySelectorAdapter');
+
+describe('IdentitySelectorAdapter', () => {
+  let sut;
+
+  beforeEach(() => {
+    sut = new IdentitySelectorAdapter();
+  });
+
+  it('should accept ipfs content type', () => {
+    // given
+    const contentType = 'ipfs';
+    // when
+    const result = sut.canHandle(contentType);
+    // then
+    expect(result).to.equal(true);
```

Comment:
you can also use `assert` from chai. `expect(x).to.equal(true)` === `assert(x)`

Reply:
but you don't have to, just hint

Reply:
Or try `expect(result).to.be.true;`